modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -937,7 +937,7 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
     return false;
   };
 
-  const isEmpty = (node: Node, elements?: Record<string, any>) => {
+  const isEmpty = (node: Node, elements?: Record<string, any>, data?: ({ includeZwsp?: boolean })) => {
```

Comment:
Maybe call it `extra` or `options`? I guess we could in v7 move elements into this options structure.

Reply:
renamed in `options`